hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1277,24 +1283,6 @@ public function upgrade_1037() {
     
     return TRUE;
   }
-
-  /**
-   * Renames option group title from job contract end reason
-   * to contract end reasons
-   *
-   * @return bool
-   */
-  public function upgrade_1041() {
```

Comment:
The function was not properly positioned after merging into the parent ticket. That is why it was refactored.